h2o-r/tests/testdir_algos/gbm/runit_GBM_quantile.R

```diff
@@ -64,7 +64,8 @@ gbm.quantile.test = function(){
   #a test to look at h2o.quantile() - init_F is equal to POJO splits for one node.
   left_q = h2o.quantile(data.hex.test.left[,resp],Q) 
   left_q_test = left_q - m@model$init_f
-  print(paste("Quantile GBM left split", left_q_test, LEFT_POJO))
+  left_q_test
+  LEFT_POJO
```

Comment:
this does nothing, does it?

Reply:
true, eventually I ended up not touching this test